### ISSUE TITLE
SnapAV Wattbox[BUG]

### ISSUE THREAD

**Issue by cbcJosh** [bitfocus/companion-module-snapav-wattbox#8]
### Is this a bug in companion itself or a module?

- [X] I believe this to be a bug in companion

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

Updated Companion to 3.2.2 and the SnapAV Wattbox does not respond to being triggered. Downgrade to 3.1.2 and everything works as expected.

### Steps To Reproduce

Upgrade to 3.2.2

### Expected Behavior

Wattbox respond to commands

### Environment (please complete the following information)

```markdown
- OS:Linux
- Browser:Chrome and Firefox
- Companion Version:3.1.2
```

**Comment by cbcJosh** [comment]
Any updates on this?

**Comment by cbcJosh** [comment]
Still wondering if there is an update on this issue?

**Comment by ryangrams** [comment]
Hey all! I'm also having trouble getting companion to talk to my WattBox. I also tried through the TCP module with no success (though that could certainly just be human error).

**Comment by josephdadams** [comment]
Are you getting any error messages?

**Comment by cbcJosh** [comment]
> Are you getting any error messages?

No error messages, just fails to toggle the power switch.

**Comment by josephdadams** [comment]
What model and protocol? Some use telnet and some use http.

**Comment by cbcJosh** [comment]
> What model and protocol? Some use telnet and some use http.

WB-300-IP-3

**Comment by josephdadams** [comment]
Samesies!!

**Comment by cookdoeyl** [comment]
This may be apart of this same bug. I'm running Companion v3.4.3. My WattBox module won't start when Polling is enabled. If I create a new instance of the module with polling disabled, it will start but not control anything.

Model: WB-300-IP-3
Firmware: WB10.B929

Module Instance Log Entry (Polling Enabled):
Starting up module class: 
Sentry disabled
error: Error getting config fields: Call timed out
error: Error getting config fields: Call timed out
error: Module registration failed Error: Call timed out
    at t.IpcWrapper.sendWithCb (C:\Program Files\Companion\resources\bundled-modules\snapav-wattbox\main.js:2:359332)
    at C:\Program Files\Companion\resources\bundled-modules\snapav-wattbox\main.js:2:352391
error: Error getting config fields: Call timed out

Module Instance Log Entry (Polling Disabled - Attempting to set Outlet 3 to Off):
error: Error executing action: Cannot read properties of undefined (reading 'get')
error: Error executing action: Cannot read properties of undefined (reading 'get')

UPDATE: I tried rolling back just the wattbox module to 2.0.1 from the bundled modules repo. I gained control back but still no polling/variables.
